modules/distribution/src/repository/resources/conf/default.json

```diff
@@ -235,5 +235,127 @@
     "org.wso2.carbon.identity.entitlement.pip.DefaultResourceFinder"
   ],
   "identity.entitlement.entitlement_engine_caching_interval": "100000",
-  "identity.entitlement.JSON_shorten_form_enabled": false
+  "identity.entitlement.JSON_shorten_form_enabled": false,
```

Comment:
why can't we use lower case 'json' here?

Reply:
Since it is a standard abbreviation I have used capital. If we need to fix we can introduce a key-mapping to maintain the backward compatibility.